src/lib/fetcher.ts

```diff
@@ -151,6 +151,8 @@ export default class Fetcher implements APIFetcher {
 
     while (true) {
       const paramsWithSkip = {
+        limit: '500',
```

Comment:
Maybe let's factor this out into a constant so we can adapt this in the future without having to manually change all these tests another time